README.md

```diff
@@ -49,13 +57,26 @@ npm start
 
 ```
 
+Run the _scholly/service_ using `npm` (Note: scholly service will need to be running concurrently in order to deploy a wesbite through _Researchify_)
+```bash
+# to run the scholly service
+cd scholly/service
+npm start
+```
+
 Or via [docker-compose](https://docs.docker.com/compose/reference/build/):
 
 ```bash
 # in root project directory
 docker-compose up
 ```
 
+To view the client website User Interface using fake data, run scholly/base using `npm`
+```bash
+cd scholly/base
```

Comment:
@kdav108 I belive you also need to export an env var to use the fake data?

Was it `REACT_APP_DEBUG`
so maybe add to this `export REACT_APP_DEBUG=true`?

Reply:
Running the command `npm run dev` on scholly/base will run the client website locally which has been set to use fake data already. Not sure if this export is needed as I don't currently have any exports for it in my `.env` file and the client website shown after running this command does contains fake data. Refer to pic below for publications page: 
![image](https://user-images.githubusercontent.com/80957318/130772484-46f2f7ac-faac-4ec7-bfe1-3c9c058d2301.png)
I've also had a look at the database to make sure that the team shown on this client website is not actually an existing team so pretty sure this command would work fine if you're looking to just see how the client website UI will look.

Reply:
Thanks, SGTM. Resolving.